README.md

```diff
@@ -59,7 +59,7 @@ Team Centri-Pedal Motion (RIT 2014) ran into an issue where there were a few Cor
 	antlib.xml:584: shellscript returned: 1`
 
 ###Required Cordova Plugins
-Below is a list of the required plugins. We have included a script that will fetch these automatically by simply running `python fetchPlugins.py` for mac, or `python fetchPluginsWindows.py` for Windows. This assumes that the locations of these repos are still as written below.
+Below is a list of the required plugins. We have included a script that will fetch these automatically by simply running `python fetchPlugins.py`. This assumes that the locations of these repos are still as written below.
```

Comment:
Since the scripts now reflect the TourTrak repos, you may want to update the links below